### PR TITLE
Sugar Factory

### PR DESCRIPTION
**Pull request by Butterspekulatius**
I hope it works now. Sorry it didn´t worked the first time.

And I´m already using Notepad++

### CODE REVIEW

**Comment by Butterspekulatius** [comment]
Can somehow delete or ignore those earlier commits?
I tried to tidy it up but now its looks like a mess.

File: CWE/common/production_types.txt [hunk 1]
```diff
@@ -1541,6 +1541,25 @@ communication_factory = {
 
 ### BEVERAGES ###
 
+sugar_factory = {       
+	template = factory_template	
+	input_goods = { 
+		fruit = 1,5
```
Comment (settintotrieste) [inline CWE/common/production_types.txt:1547]: It should be fruit = 1.5 instead of fruit = 1,5

File: CWE/common/production_types.txt [hunk 2]
```diff
@@ -1541,6 +1541,25 @@ communication_factory = {
 
 ### BEVERAGES ###
 
+sugar_factory = {       
+	template = factory_template	
+	input_goods = { 
+		fruit = 1,5	
+	}
+	output_goods = sugar
+	value = 2,6
```
Comment (settintotrieste) [inline CWE/common/production_types.txt:1550]: It should be value = 2.6 instead of value = 2,6

**Review by settintotrieste — CHANGES_REQUESTED** [review]

**Comment by settintotrieste** [comment]
I’m afraid it’s not possible to remove those commits unless you create a separate branch and then PR that one.

**Comment by settintotrieste** [comment]
If you want to add localisation, you can copy one of the entries in localisation/text.csv to create a description for sugar factory. You’ll have to open the localisation file with word pad though as notepad++ will mess with the character encoding of csv files.

**Comment by Butterspekulatius** [comment]
Ok. I change the comma into a point.

Should I create a branch to make a single PR with the changes and the loc? So its only a single commit?

And thank you for the help. I realy appreciate it.

File: CWE/localisation/text.csv [hunk 3]
```diff
@@ -22142,6 +22142,8 @@ suez_canal_treaty_annuity_desc;We are currently processing the payments from the
...
 sugar;Sugar;;;;;;;;;;;;x,,;;;;;;;;;;;;;;;;;
 sugar_RGO_SIZE;Sugar plantation size;;;;;;;x
+sugar_factory; Sugar Factory;;;;;;;;;;;;x,,;;;;;;;;;;;;;;;;;
```
Comment (settintotrieste) [inline CWE/localisation/text.csv:22145]: There's an extra space in "sugar_factory; Sugar Factory"

File: CWE/localisation/text.csv [hunk 4]
```diff
@@ -22142,6 +22142,8 @@ suez_canal_treaty_annuity_desc;We are currently processing the payments from the
...
+sugar_factory_desc; A refinery where sugar beets are processed into raw sugar.;;;;;;;;;;;;x,,;;;;;;;;;;;;;;;;;
```
Comment (settintotrieste) [inline CWE/localisation/text.csv:22146]: There's an extra space in "sugar_factory_desc; A refinery..."

**Review by settintotrieste — CHANGES_REQUESTED** [review]

**Comment by settintotrieste** [comment]
No problem and sure, you can make a separate branch to make the commits look neater.

**Comment by Butterspekulatius** [comment]
Ok, then I will change the loc and make the whole thing a seperate branch.

**Comment by settintotrieste** [comment]
Thanks!

**Pull request closed without merging**